### ISSUE TITLE
Branch name with "/" results in 500 error

### ISSUE THREAD

**Issue by Danzelot** [bast/cicero#85]
If a branch contains a "/" it can't be loaded, e.g. "bugfix/gui" results in 500.

**Comment by bast** [comment]
That's true. Good find. I need to think about how to solve this though ... but at the minimum with a better error message.
